app/views/interviews/_interview.html.slim

```diff
@@ -25,4 +25,14 @@
     td
       = link_to interview_path(i), method: :delete, data: { confirm: 'Are you sure?' }, class: "btn btn-sm btn-danger" do
         i.glyphicon.glyphicon-remove
-
+    td
+      = link_to edit_interview_interview_question_path(i, i.interview_questions.first), class: "btn btn-sm btn-info"
+        - m = 0
+        - a = 0
+        -i.interview_questions.each do |q|
+          - m +=1 if (q.mark).nil?
+          - a += 1
```

Comment:
Any kind of logic must be avoided in a view. 
This must be refactored. Consider helpers, controllers and models as possible place for such code.